distribution/ddf-common/src/main/resources/bin/service.ps1

```diff
@@ -0,0 +1,10 @@
+$psi = New-Object System.Diagnostics.ProcessStartInfo
```

Comment:
Ran into an issue where starting the Solr server left a terminal open in Windows. The only way to prevent it was to use PowerShell to start the Solr server.